lib/template.html

```diff
@@ -181,7 +181,7 @@
 {% endif %}
 
 {% if site.logo %}
-    "logo": {{ site.logo | prepend: "/" | prepend: seo_url | jsonify }},
+    "logo": {{ site.logo | prepend: seo_url | jsonify }},
```

Comment:
This is the only time that logo is used. Page image would be a better candidate for a variable as it is used 3 times with the same filters each time.